<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-4FN0AGO1" failures="0" tests="3" name="Default test" time="15.999" errors="0" timestamp="2023-03-16T14:09:25 IST">
  <testcase classname="POMPractice.FacebookLoginTestcaseExecution" name="@BeforeTest userEnterValidateEmailIdAndPassword" time="15.911">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.apache.poi.ss.usermodel.Cell.getStringCellValue()&amp;quot; because the return value of &amp;quot;org.apache.poi.ss.usermodel.Row.getCell(int)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Cell.getStringCellValue()" because the return value of "org.apache.poi.ss.usermodel.Row.getCell(int)" is null
at basic.ParameterExcelFetch.getExelData(ParameterExcelFetch.java:13)
at POMPractice.FacebookLoginTestcaseExecution.userEnterValidateEmailIdAndPassword(FacebookLoginTestcaseExecution.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest userEnterValidateEmailIdAndPassword -->
  <testcase classname="POMPractice.FacebookLoginTestcaseExecution" name="ValidateLoginFacebookWithInvaliedUsernameAndInvaliedPassword" time="0.0">
    <skipped/>
  </testcase> <!-- ValidateLoginFacebookWithInvaliedUsernameAndInvaliedPassword -->
  <testcase classname="POMPractice.FacebookLoginTestcaseExecution" name="ValidateLoginFacebookWithValiedUsernameAndInvaliedPassword" time="0.0">
    <skipped/>
  </testcase> <!-- ValidateLoginFacebookWithValiedUsernameAndInvaliedPassword -->
  <testcase classname="POMPractice.FacebookLoginTestcaseExecution" name="userEnterValidateEmailIdAndPassword" time="0.0">
    <skipped/>
  </testcase> <!-- userEnterValidateEmailIdAndPassword -->
</testsuite> <!-- Default test -->
